pkg/v2/release.go

```diff
@@ -56,8 +56,10 @@ func MapReleaseWithUnSupportedAPIs(mapOptions common.MapOptions) error {
 		return nil
 	}
 
-	log.Printf("Deprecated or removed APIs exist, updating release: %s.\n", releaseName)
-	if !mapOptions.DryRun {
+	if mapOptions.DryRun {
+		log.Printf("Deprecated or removed APIs exist, no changes will be made to release: %s.\n", releaseName)
```

Comment:
Might read better as:

Deprecated or removed APIs exist, no changes will be made to release --> Deprecated or removed APIs exist, for release

Reply:
Done